tools/cli/src/webpack/runtime-config.ts

```diff
@@ -142,6 +141,7 @@ export function getRuntimeConfig(buildFlags: BuildFlags): RuntimeConfig {
   }
 
   return {
+    isSelfHosted: process.env.SELF_HOSTED === 'true',
```

Comment:
any differences?

Reply:
`process.env.CI` always be `true` under CI, so `SELF_HOSTED` does not put to the runtime config.

https://github.com/toeverything/AFFiNE/blob/18224a83d1ed3fb96ade33f1c32140602e8b78c8/tools/cli/src/webpack/runtime-config.ts#L149